app/controllers/users_controller.rb

```diff
@@ -46,6 +46,35 @@ def check_account_creation_status
     end
   end
 
+  # Run a set of corrections on a user
+  def check
+    unless logged_in_as_admin?
```

Comment:
Should all of this code be in the admin_users_controller instead? https://github.com/otwcode/otwarchive/blob/master/app/controllers/admin/admin_users_controller.rb